components/python/dbus-python/Makefile

```diff
@@ -11,20 +11,22 @@
 #
 # Copyright 2017 Alexander Pyhalov
 # Copyright 2019 Michal Nowak
+# Copyright 2022 Gary Mills
 #
 
 BUILD_BITS=		64
+BUILD_STYLE=		setup.py
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		dbus-python
-COMPONENT_VERSION=	1.2.16
+COMPONENT_VERSION=	1.2.18
 COMPONENT_PROJECT_URL=	http://www.freedesktop.org/wiki/Software/DBusBindings/
 COMPONENT_SUMMARY=	Python bindings for D-Bus
```

Comment:
Please use https where possible.

Reply:
I've changed both of those, and verified that they worked.